libs/scully/src/lib/utils/exitHandler.ts

```diff
@@ -19,7 +20,11 @@ export function installExitHandler(): void {
 
   function exitHandler(options, exitCode) {
     for (const handler of exitHandlers) {
-      handler();
+      try {
+        handler();
+      } catch (e) {
+        logWarn(`error while closing Scully ${e.toString()}`)
```

Comment:
Maybe we could capture the error here and send it to `Sentry` so that we can gather information why it didn't closed?

Reply:
```suggestion
        logWarn(`Error while closing Scully ${e.toString()}`)
```